examples/quickstarts/helidon-quickstart-se/pom.xml

```diff
@@ -201,4 +174,31 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
```

Comment:
Health checks do not work in Graal native image as they are now.
In Graal, we would need to replace line 101 in `Main.java`:
Instead of `.add(HealthChecks.healthChecks())   // Adds a convenient set of checks`
we would need: `.add(HealthChecks.heapMemoryCheck())`
as file system health check fails on MacOS and deadlock healthcheck can work on neither environment, as this feature is not supported in Graal native image at all

Reply:
Maybe we should update implementation of these two health checks to check if in graal - and if so, just return "up" without any initialization...